src/sidebar/util/wrap-react-component.js

```diff
@@ -83,10 +89,10 @@ class ReactController {
 
   updateReactComponent() {
     // Render component, with a `ServiceContext.Provider` wrapper which
-    // provides access to Angular services via `withServices` or `useContext`
+    // provides access to services via `withServices` or `useContext`
```

Comment:
When we come to complete the migration, the top level of the component tree in `src/sidebar/index.js` will probably look something like:

```js
const container = new Injector();

// Register services...

<ServiceContext.Provider value={container}>
  <HypothesisApp/>
</ServiceContext.Provider>
```